index.js

```diff
@@ -173,3 +173,26 @@ form.addEventListener('submit', (e) => {
     e.run();
   }
 });
+
+const content = () => {
```

Comment:
[OPTIONAL] Kindly endeavour to name your functions in the [imperative](https://www.30secondsofcode.org/articles/s/javascript-naming-conventions). This is the best practice. You could say for example `saveInputs` instead of just content.